RoaringBitmap/src/main/java/org/roaringbitmap/longlong/HighLowContainer.java

```diff
@@ -54,6 +54,10 @@ public void put(byte[] highPart, Container container) {
     art.insert(highPart, containerIdx);
   }
 
+  public void remove(byte[] highPart) {
+    art.remove(highPart);
```

Comment:
If we decide to remove the  high part from the  `ART` of `HighLowContainer`, the corresponding `Container` should also be removed from the `containers`. Otherwise, they are in the inconsistent state.

Reply:
But removing the container from the `containers` is not possible(shrinking the array will cause the containerIdx to change, that's why I don't remove the empty container), maybe mark that position as null.

Reply:
Didn't notice that, thanks for pointing it out.

I think it makes sense to resize the array in `trim`, will address it in a different PR.